src/components/toast/Toast.tsx

```diff
@@ -0,0 +1,88 @@
+
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Icon from '../icon';
+
+interface Placement {
+    horizontal: 'left' | 'center' | 'right';
+    vertical: 'top' | 'center' | 'bottom';
+}
+
+export interface ToastProps extends Omit<React.HTMLProps<HTMLDivElement>, 'children' | 'content'> {
+    /** Optional CSS class name */
+    className?: string;
+    /** If true, close icon will be placed to the right */
+    closeIcon?: boolean;
```

Comment:
by curiosity, how the Toast closes he we don't have the close icon? (timeout?)

Reply:
So show props toggles visibility, onClickClose you should pass in a function that as a "suggestion" sets show to false.

I think future development of Toast will have a timeout, can be the same base logic as for Tooltip. Then we'll probably keep the show state inside the component.